.eslintrc.js

```diff
@@ -52,6 +52,7 @@ module.exports = {
     'default-param-last': 'off',
     'prefer-object-spread': 'error',
     'require-atomic-updates': 'off',
+    'arrow-body-style': 1,
```

Comment:
Can we use strings instead of numbers here? We've been trying to avoid using numbers so we don't have to remember what they mean.

Reply:
I tried `'error'` but for some reason the linting bombed, requesting a number.  I'll look into it, it was weird.

Reply:
Update: We can't have nice things: https://github.com/MetaMask/eslint-config/pull/205#issuecomment-921169164